code/modules/paperwork/records/info/he.dm

```diff
@@ -446,3 +446,96 @@
 	<h4>Insight:</h4> Common<br>
 	<h4>Attachment:</h4> Very Low<br>
 	<h4>Repression:</h4> Common<br>"}
+
+//The Watchman
+/obj/item/paper/fluff/info/he/watchman
+	name = "O-01-132"
+	info = {"<h1><center>O-01-132</center></h1>    <br>
+	Name : The Watchman    <br>
+	Risk Class : HE    <br>
+	Max PE Boxes : 18    <br>
+	Qliphoth Counter : 2    <br>
+	Work Damage Type : Black	<br>
+	Work Damage : Moderate	<br>
+	- When the work result was Bad, the Qliphoth Counter decreased.    <br>
+	- When the work result was Neutral, the Qliphoth counter lowered at a moderate chance. <br>
+	- When employees with Temperance Level 3 or higher worked on The Watchman, the work rates were reduced.<br>
+	- While breached, The Watchman flooded the area around with darkness. Said darkness has the same propogation as light.<br>
+	<br>
+	<h4>Instinct:</h4> Common<br>
+	<h4>Insight:</h4> High<br>
+	<h4>Attachment:</h4> Common<br>
+	<h4>Repression:</h4> Very Low<br>
+	<h4>Escape Damage:</h4> None	 <br>
+	<h4>Red Resistance:</h4> Normal	<br>
+	<h4>White Resistance:</h4> Weak	<br>
+	<h4>Black Resistance:</h4> Endured	<br>
+	<h4>Pale Resistance:</h4> Fatal	<br>"}
+
+//Schadenfreude
+/obj/item/paper/fluff/info/he/schadenfreude
+	name = "O-05-76"
+	info = {"<h1><center>O-05-76</center></h1>    <br>
+	Name : Schadenfreude    <br>
+	Risk Class : HE    <br>
+	Max PE Boxes : 18    <br>
+	Qliphoth Counter : 4    <br>
+	Work Damage Type : Red	<br>
+	Work Damage : Moderate	<br>
+	- Work with Schadenfreude must proceed while the Containment Unit is viewable by only one person.    <br>
+	- When Schadenfreude was in the view of more than 2 people, work rates decreased.  <br>
+	- When Schadenfreude was in the view of more than 3 people, the Qlipoth Counter decreased.  <br>
+	- Schadenfreude became completely stationary when it was only viewed by one person. Please take this into account when suppressing it. <br>
+	<br>
+	<h4>Instinct:</h4> Very Low<br>
+	<h4>Insight:</h4> Common<br>
+	<h4>Attachment:</h4> Common<br>
+	<h4>Repression:</h4> Common<br>
+	<h4>Escape Damage Type:</h4> Red	<br>
+	<h4>Escape Damage:</h4> High	 <br>
+	<h4>Red Resistance:</h4> Endured	<br>
+	<h4>White Resistance:</h4> Resistant <br>
```

Comment:
Resistance Notation
Absorbs: -X
Immune: 0
Ineffective: 0.1-0.4
Endured: 0.5-0.9
Normal: 1
Weak: 1.1-1.5
Fatal: 1.6-2

Reply:
We spoke about this; and decided to change it all later